### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by maksym-nemera**

### CODE REVIEW

**Review by DidenkoDenis306 — APPROVED** [review]
Good job!

File: src/inverseRobot.js [hunk 1]
```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const value in robot) {
+    const keys = robot[value];
```
Comment (naziamoff) [inline src/inverseRobot.js:13]: This confused me a little. This is a single value, but from its name it seems it's in plural.
```suggestion
    const key = robot[value];
```

File: src/inverseRobot.js [hunk 2]
```diff
@@ -7,7 +7,19 @@
...
+
+    if (Object.keys(newRobot).includes(keys)) {
+      return null;
+    }
```
Comment (naziamoff) [inline src/inverseRobot.js:17]: not bad, but same as in my comment above:
```suggestion
    if (Object.keys(newRobot).includes(key)) {
      return null;
    }
```

**Review by naziamoff — APPROVED** [review]
Looks good 👍

**Pull request closed without merging**